cmd/cmd_utils.go

```
@@ -543,3 +544,91 @@ func CheckForAtmosUpdateAndPrintMessage(atmosConfig schema.AtmosConfiguration) {
+				u.PrintMessage(fmt.Sprintf("  %s", sub))
+			}
+		} else {
+			u.PrintMessage("No valid subcommands found")
+		}
+	}
+	u.PrintMessage(fmt.Sprintf(`Run '%s --help' for usage`, cmd.CommandPath()))
+	u.LogErrorAndExit(atmosConfig, errors.New(unkonwnCommand))
+}
+
+// hasPositionalArgs checks if a slice of strings contains an exact match for the target string.
+func hasPositionalArgs(args []string) bool {
```

golangci-lint (1.62.2)

func `hasPositionalArgs` is unused

(unused)